test/evergreen.yml

```diff
@@ -2225,8 +2205,12 @@ tasks:
             set -o errexit
             set -o verbose
             # ${test_env_vars|} $(pwd)/../test/csuite/random_abort/smoke.sh 2>&1
-            ${test_env_vars|} $(pwd)/../test/csuite/timestamp_abort/smoke.sh 2>&1
-            ${test_env_vars|} $(pwd)/test/csuite/test_truncated_log 2>&1
+      - func: "csuite smoke test"
+        vars:
+          test_binary: timestamp_abort
+      - func: "csuite smoke test"
+        vars:
+          test_binary: truncated_log
```

Comment:
Any reason the `test_` prefix is removed?

Reply:
Yes, for smoke scripts, we are looking into the directory `test/csuite/<test_name>/smoke.sh`.
For the binaries, we are looking into: `<build_dir>/test/csuite/test_<test_name>`.